plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -378,8 +384,16 @@ function MosaicMenuItem:init()
             style = self.shortcut_style,
         }
     end
+
+    -- self.progress_bar = ProgressWidget:new
+
+
     self.detail = self.text
 
+    if not self.menu.cover_info_cache then
+        self.menu.cover_info_cache = {}
+    end
```

Comment:
In MosaicMenuItem:update(), at the same place it is in ListMenuItem, to keep both consistent.